Makefile

```diff
@@ -0,0 +1,19 @@
+build: migrate.up
+	go build main.go
+
+migrate.up:
+	cp .env.example .env
+	cd db
+        include .env
```

Comment:
can you shift lines 7 to 9 to the top of the makefile?

Reply:
You mean move it to `init`? Migrate.up will run first before build.

Reply:
nope, i think its better to move the 3 lines to the top of the Makefile, so other commands can use the variables declared as well

can probably also remove the copying of .sample.env to .env --> to allow users to manually configure their own .env file themselves

Reply:
Thanks for telling that we can do no-target prerequisities before first target in `Makefile`. I've tried it, but it doesn't work if the command is `echo`.

Reply:
> remove the copying of .sample.env to .env ...

About that, maybe we need different header talking about env.

Reply:
this works for me
![image](https://github.com/jeyren95/big-bang-theory-quotes/assets/65485512/ce09113b-89cd-493f-8b2a-8ed808162da1)

Reply:
In my machine, it doesn't work if the command is `echo`

Reply:
No, we didn't need remove ```cp .env.example .env```, user can just edit the example as they wish. I think it'll better if we add `Setting up environment variable (optional)` heading instead in `README.md`

Reply:
you mean like this?
![image](https://github.com/jeyren95/big-bang-theory-quotes/assets/65485512/6cc579ac-de96-4f41-9d35-71c2e6c33fa6)

this works for me though haha

Reply:
but if we dont remove cp .env.example .env, then when the user runs "make migrate.up", the command will automatically use the default env variables declared in .env.example

Reply:
which is not what we want? we want to allow the user to use their own "SQLITE_DSN" env variable as desired

Reply:
I mean like this.
![image](https://github.com/jeyren95/big-bang-theory-quotes/assets/63584188/c49a0d7d-f417-4db5-b4af-02e14a3c9f93)

Reply:
I mean like this. 

![image](https://github.com/jeyren95/big-bang-theory-quotes/assets/63584188/a9f79a3e-7c1e-4664-b0e8-dfe5abcae1aa)

Reply:
> we want to allow the user to use their own "SQLITE_DSN" env variable as desired
About that, sure I can do that.

Reply:
Sure, temporal conclusion in this conversion:
- Remove `cp .env.example .env`, rename `.env.example` to `.env`.
- Move line 7-9 to the top in `Makefile`
- Add `.env` documentation in `README.md` before `make build`

Is it good enough?

Reply:
just remove cp .env.example .env, dont have to rename

the rest is good

Reply:
Oh sorry, just realized there's already default `.env`. Because there isn't any clarification, i'll working on it.